src/issues.js

```diff
@@ -435,25 +435,40 @@ export default class JiraIssues {
   /**
   * Make issue transition
   */
-  async makeTransition( issueId ){
+  async makeTransition( issueId, transitionName ){
 
     const transitions = await this.getIssueTransitions(issueId);
-    const _this = this;
 
-    if ( transitions.length == 1 ) {
+    let transitionObj;
+
+    if (transitionName) {
```

Comment:
@dancrumb excellent! just one little detail, can you please add an additional check here, the transitionName is returning `true` if there is no parameter, so we need to introduce something like `if (transitionName && transitionName !== true) {` what do you think?

Reply:
I think this will do it

```suggestion
    if (typeof transitionName === 'string' && transitionName.length > 0) {
```